emc/dmri.py

```diff
@@ -0,0 +1,96 @@
+"""Representing data in hard-disk and memory."""
+import attr
+import numpy as np
+import h5py
+from pathlib import Path
+
+
+@attr.s(slots=True)
+class DWI:
+    """Data representation structure for dMRI data."""
+
+    dataobj = attr.ib(default=None)
```

Comment:
Does this name follow a specific convention? Maybe better to call it `dwdata` to be more explicit that these are the diffusion weighted images?

Reply:
nibabel gives access to the numpy array through `dataobj`, but it is not required, honestly.